### PR TITLE
[10.0][FIX] Fix the module adding the street3

### PR DESCRIPTION
**Pull request by giangtnm**
Remove empty lines which can happen when street3 field is empty.

### CODE REVIEW

File: partner_address_street3/model/partner.py [hunk 1]
```diff
@@ -17,3 +17,12 @@ def _address_fields(self):
...
         fields.append('street3')
         return fields
+
+    @api.multi
+    def _display_address(self, without_company=False):
+        """Remove empty lines which can happen when street3 field is empty."""
+        res = super(ResPartner, self)._display_address(
+            without_company=without_company)
+        while '\n\n' in res:
+            res = res.replace('\n\n', '\n')
```
Comment (yajo) [inline partner_address_street3/model/partner.py]: I think a regexp would do a better job here.

**Review by yajo — APPROVED** [review]

File: partner_address_street3/model/partner.py (hunk 1 above)
Comment (giangtnm) [reply partner_address_street3/model/partner.py]: @Yajo I changed it to regex.

File: partner_address_street3/model/partner.py [hunk 2]
```diff
@@ -23,6 +23,5 @@ def _display_address(self, without_company=False):
         """Remove empty lines which can happen when street3 field is empty."""
         res = super(ResPartner, self)._display_address(
             without_company=without_company)
-        while '\n\n' in res:
-            res = res.replace('\n\n', '\n')
+        res = re.sub(r'([\n]+)', r'\n', res)
```
Comment (yajo) [inline partner_address_street3/model/partner.py]: Shouldn't this be `r'\n{2,}'`?

Comment (giangtnm) [reply partner_address_street3/model/partner.py]: @Yajo Thank you, I changed it.

**Review by jcdrubay — APPROVED** [review]

**Comment by jcdrubay** [comment]
@OCA/core-maintainers : could this be merged? thanks :)

**Comment by pedrobaeza** [comment]
Please change module version number according https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#version-numbers

**Review by NL66278 — APPROVED** [review]
:+1: LGTM

**Pull request merged**